apps/src/templates/sectionAssessments/FreeResponseBySurveyQuestionContainer.jsx

```diff
@@ -24,7 +34,11 @@ class FreeResponseBySurveyQuestionContainer extends Component {
         <h2>{i18n.studentFreeResponseAnswers()}</h2>
         {freeResponsesByQuestion.map((question, index) => (
           <div key={index}>
-            <h3>{`${question.questionNumber}. ${question.questionText}`}</h3>
+            <div style={styles.text}>
+              <span>{`${question.questionNumber}. ${question.questionText}`.slice(0, QUESTION_CHARACTER_LIMIT)}</span>
+              {((question.questionText.length >= QUESTION_CHARACTER_LIMIT))
```

Comment:
I think it's a bit cleaner to write this line like this:
```
{ question.questionText.length >= QUESTION_CHARACTER_LIMIT &&
  <a href="#"><span>{i18n.seeFullQuestion()}</span></a>
}
```

Reply:
Fixed, thanks!